### PR TITLE
Split main project into smaller ones

### PR DESCRIPTION
**Pull request by sqreek**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,26 +15,28 @@ repositories {
...
+//    compile 'com.theoryinpractise:halbuilder-standard:4.0.1'
+//
+//    compile('com.theoryinpractise:halbuilder-jaxrs:1.1.1') {
+//        exclude group: 'javax.ws.rs'
+//    }
+//
+//
+//    testCompile 'junit:junit:4.11'
+//    testCompile 'io.dropwizard:dropwizard-testing:0.7.1'
+//    testCompile 'org.mockito:mockito-core:1.9.5'
```
Comment (sqreek) [inline build.gradle]: self comment hi five: remove commented code

File: pandora-api/build.gradle [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+sourceCompatibility = 1.8
+version = '1.0'
+
+
+repositories {
+    jcenter()
+    mavenCentral();
+
+    maven {
+        url "http://repo.surech.ch/content/repositories/hosted.halarious.releases/"
```
Comment (nandy-andy) [inline pandora-api/build.gradle]: We don't need it if we don't use halarious

File: pandora-api/build.gradle [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+}
+
+// In this section you declare the dependencies for your production and test code
+dependencies {
+    // The production code uses the SLF4J logging API at compile time
+    compile 'org.slf4j:slf4j-api:1.7.5'
+    compile 'io.dropwizard:dropwizard-core:0.7.1'
+    compile 'io.dropwizard:dropwizard-client:0.7.1'
+    compile 'ch.halarious:halarious-core:0.11'
```
Comment (nandy-andy) [inline pandora-api/build.gradle]: Remove it since we went with halbuilder :)

**Comment by drsnyder** [comment]
Please add a [description](https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md).

**Pull request merged**